Core/HLE/sceNet.cpp

```diff
@@ -457,6 +466,22 @@ static int sceNetUpnpGetNatInfo()
 	return 0;
 }
 
+static int sceNetGetDropRate(u32 dropRateAddr, u32 dropDurationAddr)
+{
+	INFO_LOG(SCENET, "sceNetGetDropRate %08x,%08x", dropRateAddr, dropDurationAddr);
+	Memory::Write_U32(netDropRate, dropRateAddr);
+	Memory::Write_U32(netDropDuration, dropDurationAddr);
+	return 0;
```

Comment:
Can you use the new logging for this?  It will look like this:

```c++
static int sceNetGetDropRate(u32 dropRateAddr, u32 dropDurationAddr) {
	Memory::Write_U32(netDropRate, dropRateAddr);
	Memory::Write_U32(netDropDuration, dropDurationAddr);
	return hleLogSuccessInfoI(SCENET, 0);
}
```

This will automatically log the addresses AND the values.  It uses the parameter values in the table I made my other comment on to decide how to format.

-[Unknown]